client/src/state/editor/addElementMutation.js

```diff
@@ -0,0 +1,42 @@
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+// GraphQL query for deleting a specific block
+const mutation = gql`
+mutation AddBlock($className: String!, $elementalAreaID: ID!, $afterElementID: ID) {
```

Comment:
This mutation should be called `AddElement` or maybe `AddElementToArea`.

Reply:
Yes, it should.